ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py

```diff
@@ -158,103 +158,103 @@ def test_torch_binary_cross_entropy(
 
 
 # binary_cross_entropy_with_logits
-@handle_frontend_test(
-    fn_tree="torch.nn.functional.binary_cross_entropy_with_logits",
-    dtype_and_true=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
-        min_value=0.0,
-        max_value=1.0,
-        large_abs_safety_factor=2,
-        small_abs_safety_factor=2,
-        safety_factor_scale="linear",
-        allow_inf=False,
-        exclude_min=True,
-        exclude_max=True,
-        min_num_dims=1,
-        max_num_dims=1,
-        min_dim_size=2,
-    ),
-    dtype_and_pred=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
-        min_value=1.0013580322265625e-05,
-        max_value=1.0,
-        large_abs_safety_factor=2,
-        small_abs_safety_factor=2,
-        safety_factor_scale="linear",
-        allow_inf=False,
-        exclude_min=True,
-        exclude_max=True,
-        min_num_dims=1,
-        max_num_dims=1,
-        min_dim_size=2,
-    ),
-    dtype_and_weight=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
-        min_value=1.0013580322265625e-05,
-        max_value=1.0,
-        allow_inf=False,
-        min_num_dims=1,
-        max_num_dims=1,
-        min_dim_size=2,
-    ),
-    size_average=st.booleans(),
-    reduce=st.booleans(),
-    reduction=st.sampled_from(["mean", "none", "sum", None]),
-    dtype_and_pos_weight=helpers.array_or_none(
-        array_dtype="float",
-        min_value=0,
-        max_value=10,
-        allow_inf=False,
-        exclude_min=True,
-        exclude_max=True,
-        min_num_dims=1,
-        max_num_dims=1,
-        min_dim_size=2,
-    ),
-)
-def test_torch_binary_cross_entropy_with_logits(
-    *,
-    dtype_and_true,
-    dtype_and_pred,
-    dtype_and_weight,
-    size_average,
-    reduce,
-    reduction,
-    dtype_and_pos_weight,
-    as_variable,
-    with_out,
-    num_positional_args,
-    native_array,
-    on_device,
-    fn_tree,
-    frontend,
-):
-    pred_dtype, pred = dtype_and_pred
-    true_dtype, true = dtype_and_true
-    weight_dtype, weight = dtype_and_weight
-    pos_weight_dtype, pos_weight = dtype_and_pos_weight
-    helpers.test_frontend_function(
-        input_dtypes=[
-            pred_dtype[0],
-            true_dtype[0],
-            weight_dtype[0],
-            pos_weight_dtype[0],
-        ],
-        as_variable_flags=as_variable,
-        with_out=with_out,
-        num_positional_args=num_positional_args,
-        native_array_flags=native_array,
-        frontend=frontend,
-        fn_tree=fn_tree,
-        on_device=on_device,
-        input=pred[0],
-        target=true[0],
-        weight=weight[0],
-        size_average=size_average,
-        reduce=reduce,
-        reduction=reduction,
-        pos_weight=pos_weight[0],
-    )
+# @handle_frontend_test(
```

Comment:
is this test commented out originally? if not please avoid changes outside your selected function, thank you!

Reply:
I fix the conflict with the latest master which fixes the test error for bce.

Reply:
I've fixed this one by resolving the conflicts with the latest master branch. Turns out the testing bug in BCE loss have been fixed. @juliagsy I would like my latest pushes be reviewed.